test/Altinn.Platform.Authentication.Tests/Mocks/SystemUserServiceMock.cs

```diff
@@ -57,7 +86,61 @@ public Task<int> SetDeleteFlagOnSystemUser(Guid systemUserId)
         /// <returns></returns>
         public Task<int> UpdateSystemUserById(SystemUserUpdateDto request)
         {
-            throw new System.NotImplementedException();
+            int array = theMockList.FindIndex(su => su.Id == request.Id.ToString());
+            theMockList[array].IntegrationTitle = request.IntegrationTitle;
+            theMockList[array].ProductName = request.ProductName;
+            return Task.FromResult(1);
+        }
+
+        /// <summary>
+        /// Helper method during development, just some Mock data.
+        /// </summary>
+        /// <returns></returns>        
+        private static List<SystemUser> MockDataHelper()
```

Comment:
Is there any advantage og hardcoding it as a method over using a json file for the mockdata? If json file, probably it is more readable and data kept separately

Reply:
will expand to json file in future PR